### DISCUSSION TITLE
Add support for "Display Name" for user profiles

### DISCUSSION THREAD

**Discussion by brootle** [deso-protocol/dips#227] [Medium CIPs]
Add support for "Display Name" for user profiles.

This should be important for accounts that want their profile to be more official and that's also good for SEO. A lot of benefits really.

![image](https://user-images.githubusercontent.com/20090030/154675832-60650955-01d1-4716-8f6b-71f2812a8137.png)

cc @diamondhands0

**Comment by tijno** [comment]
This is already an existing dip - https://github.com/deso-protocol/dips/discussions/24
Maybe close this and add to the existing discussion?

**Reply by brootle** [reply]
I am not sure, this DIP seems more straight forward and easy to implement

**Comment by oO** [comment]
Agreed that a simple, single field called "Display Name" is better than LastName+FirstName, which is very western, and even then, doesn't work for people who use a middle name, made-up names, entities or corporations, non-western name formats like China and Korea, and implies a verified identity like FB or Google.

As a note, Supernovas.app already supports a Display Name field. they just don't store it on-chain. 
![Screen Shot 2022-02-22 at 10 52 30 AM](https://user-images.githubusercontent.com/181864/155200244-899ca9ef-9ed2-4f1b-ad00-a09e91673141.png)

**Reply by brootle** [reply]
this is cool, I am sure Supernovas would be happy if they could store display name on chain as part of user profile 👍
